### DISCUSSION TITLE
SnapKey Error When Executed From Search Bar

### DISCUSSION THREAD

**Discussion by PhiDelta-t** [cafali/SnapKey#102] [General]
When I run snapkey from windows search an error appears like this.

![image](https://github.com/user-attachments/assets/76075003-080a-40b1-a6a0-ab40f13c8d5b)

I have replaced "backup.snapkey" in the "meta" folder but the result is the same.

**Comment by cafali** [comment]
Make sure to create a SnapKey (SnapKey.exe) shortcut and place it in "C:\ProgramData\Microsoft\Windows\Start Menu\Programs" This will launch SnapKey from the Start Menu, executing it directly from the main SnapKey folder and ensuring all necessary files are included.

**Reply by PhiDelta-t** [reply]
Thx, its works
